com.github.virtual_puppet_project.vpuppr.yml

```
@@ -0,0 +1,47 @@
+id: com.github.virtual_puppet_project.vpuppr
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: OpenSeeFaceGD.x86_64
+finish-args:
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--share=network" # to set up the deps for openseeface
+    - "--device=all"
+    - "--socket=pulseaudio"
+    - "--filesystem=home:ro"
+modules:
+
+  # --- Puppeteer ---
+  - name: puppeteer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/{resources,OpenSeeFaceFolder}
+      - install -Dm 644 -t /app/share/applications com.github.virtual_puppet_project.vpuppr.desktop
+      - install -Dm 644 com.github.virtual_puppet_project.vpuppr-128.png /app/share/icons/hicolor/128x128/apps/com.github.virtual_puppet_project.vpuppr.png
+      - mkdir -p /app/share/icons/128x128/apps/
+
+      - mv * /app/bin/
+      - mv /app/bin/gui /app/bin/licenses /app/bin/scripts /app/bin/translations /app/bin/resources/
+      - mv /app/bin/OpenSeeFace /app/bin/OpenSeeFaceFolder/
+
+    sources:
+      - type: archive
+        url: https://github.com/virtual-puppet-project/vpuppr/releases/download/0.8.1/openseeface-gd_0.8.1_linux.zip
+        sha256: 31932f233a496054a3305033406e6f3fa2b14dc54e0d2fda3895349980714735
+      - type: file
+        path: com.github.virtual_puppet_project.vpuppr.desktop
+#      - type: file
+#        path: icons/com.github.virtual_puppet_project.vpuppr-16.png
+#      - type: file
+#        path: icons/com.github.virtual_puppet_project.vpuppr-24.png
+#      - type: file
+#        path: icons/com.github.virtual_puppet_project.vpuppr-32.png
+#      - type: file
+#        path: icons/com.github.virtual_puppet_project.vpuppr-48.png
+#      - type: file
+#        path: icons/com.github.virtual_puppet_project.vpuppr-64.png
+      - type: file
+        path: icons/com.github.virtual_puppet_project.vpuppr-128.png
+#      - type: file
+#        path: icons/com.github.virtual_puppet_project.vpuppr-256.png
```

I would have kept this one (I was really taling about having the < 128px here)

```suggestion
      - type: file
        path: icons/com.github.virtual_puppet_project.vpuppr-256.png
```